lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -45,12 +42,6 @@
         class="center-block"
         path="{{ main.selection.getImageLogo() || '../../../assets/image.svg' }}"
         ng-disabled="main.shouldImageStepBeDisabled()"></svg-icon>
-      <span
-        class="icon-caption"
-        ng-disabled="main.shouldImageStepBeDisabled()">Select Image</span>
-      <span
-        class="badge space-top-medium"
```

Comment:
A quick search revealed that `space-top-medium` is not longer used. Can we delete it?